### DISCUSSION TITLE
Nvm

### DISCUSSION THREAD

**Discussion by TankPain** [FunkinCrew/Funkin#1531] [General]
Question: can u send the week 7 sources code please

**Comment by Styloboi** [comment]
I think he'll do it after it's released for download

**Comment by SilverT4** [comment]
Source code will be released once the Newgrounds page for FNF reaches 1st or 2nd result on Google search.

(Source: [Tweet](https://twitter.com/ninja_muffin99/status/1385290523955585026))

**Comment by GabrielSomething** [comment]
why would you even wanna port the game anyways?

**Comment by foxsouns** [comment]
> why would you even wanna port the game anyways?

it's not really about "porting"; this project is supposed to be open source, something that everyone could build upon, and we can't build onto it if the owner of it is keeping the source closed. ninja either needs to change the license this thing is under, or clarify some stuff, because this game's source code is becoming less and less "open".

**Comment by SilverT4** [comment]
ninjamuffin is purposely keeping the source code from us until we reach a
certain goal on google. even after the full game launches, it'll still be
open source.

**Comment by foxsouns** [comment]
> ninjamuffin is purposely keeping the source code from us until we reach a certain goal on google. even after the full game launches, it'll still be open source.
 
That's what I'm hoping, but the new kickstarter is worrying me. I just hope a decent chunk of it _stays_ open source with the release of the "Whole Ass Game".

**Comment by TankPain** [comment]
Oki thanks guys
